src/blackboxTests/src/test/kotlin/org/vaccineimpact/api/blackboxTests/tests/BurdenEstimates/PopulateBurdenEstimateTests.kt

```diff
@@ -158,4 +158,20 @@ class PopulateBurdenEstimateTests : BurdenEstimateTests()
             JSONValidator().validateError(resultAsString, expectedErrorCode = "csv-unexpected-header")
         }
     }
+
+    @Test
+    fun `cannot populate burden estimate set if duplicate rows`()
+    {
+        TestUserHelper.setupTestUser()
+        val setId = JooqContext().use {
+            setUpWithBurdenEstimateSet(it)
+        }
+        val token = TestUserHelper.getToken(requiredWritePermissions, includeCanLogin = true)
+        val helper = RequestHelper()
+        val response = helper.post("$setUrl/$setId/", duplicateCsvData, token = token)
+        JSONValidator()
+                .validateError(response.text, "duplicate-key:burden_estimate_set, country, year, age, burden_outcome")
```

Comment:
I don't mind having commas in our machine readable error codes, but the spaces are a bit unpleasant. Would it be straightforward to remove them?

Reply:
Sure, we could do that in the `PostgresErrorHandler`